admin/timescale/updates/2022-10-10-spotify-cache-normalized-1.sql

```diff
@@ -0,0 +1,56 @@
+BEGIN;
+
+CREATE SCHEMA spotify_cache;
```

Comment:
Forgot to do it earlier but will copy these statements to create_tables.sql etc after review.